tests/Cache/SerializableChecker.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Illuminate\Tests\Cache;
+
+use Serializable;
+
+class SerializableChecker implements Serializable
+{
+    public $wasSerialized = false;
+
+    public function serialize()
+    {
+        return serialize('');
+    }
+
+    public function unserialize($serialized)
+    {
+        $this->wasSerialized = true;
+    }
+}
```

Comment:
Put this class in the same test file if it's not used elsewhere.

Reply:
Good point, done!